src/applications/facility-locator/constants/index.js

```diff
@@ -123,3 +123,5 @@ export const MAX_SEARCH_AREA = 500;
  * Min radius search area in miles
  */
 export const MIN_RADIUS = 10;
+
+export const Covid19Vaccine = 'Covid19Vaccine';
```

Comment:
Our naming conventions for constants is all caps, snake case.

Reply:
P.S. I realize we aren't following this consistently, but we should. We can do it later.